org-roam.el

```diff
@@ -998,7 +998,8 @@ Applies `org-roam-link-current' if PATH corresponds to the
 currently opened Org-roam file in the backlink buffer, or
 `org-roam-link-face' if PATH corresponds to any other Org-roam
 file."
-  (cond ((not (file-exists-p path))
+  (cond ((or (file-remote-p path) ;; Prevent lockups opening Tramp links
```

Comment:
```suggestion
  (cond ((and (not (file-remote-p path)) ;; Prevent lockups opening Tramp links
```

This would prevent font-locking all remote paths as invalid links, and default to the `org-link` face.